### PR TITLE
TurtleBot3 Burger Robot

### PR DESCRIPTION
**Pull request by DavidMansolino**
Addition of the TurtleBot3 Burger robot.

https://www.cyberbotics.com/doc/guide/turtlebot3-burger?version=feature-turtelbot3
https://www.cyberbotics.com/doc/guide/robots?version=feature-turtelbot3

### CODE REVIEW

**Comment by DavidMansolino** [comment]
empty_3

First draft with rough importation with minor optimization and improvement. Still a lot of improvement to be done but already looking better than the original model :wink: 
Capture du 2019-11-29 17-03-59

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: projects/appearances/protos/RoughPolymer.proto [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#VRML_SIM R2020a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# A rough polymer material.
+
+PROTO RoughPolymer [
+  field SFColor baseColor        1 1 1  # Defines the base color of the materil.
```
Comment (stefaniapedrazzi) [inline projects/appearances/protos/RoughPolymer.proto]:
```suggestion
  field SFColor baseColor        1 1 1  # Defines the base color of the material.
```

**Comment by DavidMansolino** [comment]
> We should not add the mountains PNG and HDR images to the guide/scene/<robot>/textures folder because the default ones are used.

Thank you, I didn't knew about this (or probabyl forget), we should probably improve our script that generates the web-component to include this (I will open an issue).

**Comment by DavidMansolino** [comment]
:+1:

**Pull request merged**